<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_oeuvre</class>
 <widget class="QDialog" name="gestion_oeuvre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1911</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="qrcode">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>670</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    border: 2px solid #792382;
    background-color: white; /* Fond blanc */
    color: #792382; /* Écriture en violet */
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #43aff7;
}
</string>
   </property>
   <property name="text">
    <string>Qr code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>730</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    border: 2px solid #792382;
    color: white;
    border-radius: 5px;
    background-image: url(&quot;images/log-out.svg&quot;);
    background-repeat: no-repeat;
    background-position: left center;
    padding-left: 30px;
    font-weight: bold; /* Texte en gras */
}

QPushButton:hover {
    background-color: #43aff7;
}
</string>
   </property>
   <property name="text">
    <string>Se deconnecter</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="asc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>asc</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="desc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>desc</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>375</width>
     <height>327</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 18px;
    color: #792382; /* Couleur du texte */
    font-weight: bold; /* Gras */
    /* Ajoutez d'autres styles selon vos besoins */
}
</string>
      </property>
      <property name="text">
       <string>Nom Artiste</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nom">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 5px;
    font-size: 14px;
    color: #333;
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
      </property>
      <property name="placeholderText">
       <string>NomArtiste</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 18px;
    color: #792382; /* Couleur du texte */
    font-weight: bold; /* Gras */
    /* Ajoutez d'autres styles selon vos besoins */
}
</string>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="type">
      <property name="styleSheet">
       <string notr="true">    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 5px;
    font-size: 14px;
    color: #333;</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Peinture a l'huile</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Peinture a l'eau</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Photographe</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 18px;
    color: #792382; /* Couleur du texte */
    font-weight: bold; /* Gras */
    /* Ajoutez d'autres styles selon vos besoins */
}
</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="prix">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 5px;
    font-size: 14px;
    color: #333;
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
      </property>
      <property name="placeholderText">
       <string>Prix</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/AppData/Local/Temp/b6651d8d-1e9f-45ec-8183-973f3ea7bf13_interface à bechir.zip.f13/PROJET C++/back.jpg</pixmap>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_16">
      <property name="styleSheet">
       <string notr="true">QLabel {
    font-size: 18px;
    color: #792382; /* Couleur du texte */
    font-weight: bold; /* Gras */
    /* Ajoutez d'autres styles selon vos besoins */
}
</string>
      </property>
      <property name="text">
       <string>Evenement</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="evenement">
      <property name="styleSheet">
       <string notr="true">    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 5px;
    font-size: 14px;
    color: #333;</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="ajouter">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    border: 2px solid #792382;
    background-color: white; /* Fond blanc */
    color: #792382; /* Écriture en violet */
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #43aff7;
}
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="Annuler">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    border: 2px solid #792382;
    background-color: white; /* Fond blanc */
    color: #792382; /* Écriture en violet */
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #43aff7;
}
</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="historique">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>670</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    border: 2px solid #792382;
    background-color: white; /* Fond blanc */
    color: #792382; /* Écriture en violet */
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #43aff7;
}
</string>
   </property>
   <property name="text">
    <string>Historique</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    color: #792382; /* Couleur du texte */
    font-weight: bold; /* Gras */
    /* Ajoutez d'autres styles selon vos besoins */
}
</string>
   </property>
   <property name="text">
    <string>Gestion des Oeuvres</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    background-image: url(&quot;images/logo.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bechi/Documents/interface à bechir/interface ├á bechir/images/logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>460</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bechi/AppData/Local/Temp/Downloads/edit.svg</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1480</x>
     <y>170</y>
     <width>411</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bechi/AppData/Local/Temp/Downloads/edit.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>500</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bechi/AppData/Local/Temp/Downloads/edit.svg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadPDF">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>670</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    border: 2px solid #792382;
    color: #792382;
    border-radius: 5px;
    background-image: url(&quot;images/download.svg&quot;);
    background-repeat: no-repeat;
    background-position: left center;
    padding-left: 30px; /* Ajustez ceci selon la taille de votre image */
    background-color: #ffffff; /* Arrière-plan blanc */
}

QPushButton:hover {
    background-color: #43aff7; /* Arrière-plan bleu au survol */
    color: #ffff; /* Texte en blanc au survol */
}</string>
   </property>
   <property name="text">
    <string>Télécharger</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>330</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bechi/AppData/Local/Temp/Downloads/search (1).svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>330</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/bechi/AppData/Local/Temp/Downloads/search (1).svg</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 5px;
    font-size: 14px;
    color: #333;
    padding-left: 30px; /* Ajustez ceci selon la taille de votre image */
    background-image: url(&quot;images/search.svg&quot;);
    background-repeat: no-repeat;
    background-position: left center;
}

QLineEdit::placeholder {
    color: #666; /* Couleur du texte de repère */
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Rechercher..</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableW">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>1061</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>2631</width>
     <height>1041</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(&quot;images/back.jpg&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="triCombo">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Style de base du QComboBox */
QComboBox {
    padding: 8px;
    font-size: 16px;
    border: 2px solid #43aff7; /* Couleur de la bordure en bleu */
    border-radius: 5px;
    background-color: #fff;
    color: #792382; /* Couleur du texte en violet */
    background-image: url(&quot;C:/Users/user/Documents/interface à bechir/images/chevron-down.svg&quot;);
    background-repeat: no-repeat;
    background-position: right center; /* Position de l'image à droite et centrée verticalement */
    padding-right: 25px; /* Ajustement de la taille pour laisser de l'espace à l'image */
}

/* Style de la flèche du QComboBox */
QComboBox::drop-down {
    border: none;
    width: 20px;
    background-color: transparent;
}

/* Style de la flèche au survol */
QComboBox::drop-down:hover {
    background-color: #f0f0f0; /* Couleur de fond au survol en gris clair */
}

/* Style du QComboBox au survol */
QComboBox:hover {
    background-color: #f0f0f0; /* Couleur de fond au survol en gris clair */
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Trier Selon :</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prix</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Etat</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="stat">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    border: 2px solid #792382;
    color: #792382; /* Changement de couleur du texte en violet */
    border-radius: 5px;
    background-image: url(&quot;images/bar-chart.svg&quot;);
    background-repeat: no-repeat;
    background-position: left center;
    padding-left: 30px; /* Ajustez ceci selon la taille de votre image */
    background-color: #fff; /* Changement de couleur de fond en blanc */
}

QPushButton:hover {
    background-color: #43aff7; /* Changement de couleur de fond au survol */
    color: #792382; /* Garder la couleur du texte en violet */
}

</string>
   </property>
   <property name="text">
    <string>Statistique</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>1551</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="statOFF">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-image: url(&quot;images/x.svg&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    color: #792382; /* Couleur du texte */
    border-radius: 5px;
    background-color: #ffffff; /* Arrière-plan blanc */
    border: none; 
    background-image: url(&quot;images/refresh-ccw.svg&quot;); 
    background-repeat: no-repeat; /* Ne pas répéter l'image */
    background-position: center; /* Centrer l'image */
}

QPushButton:hover {
    background-color: #43aff7; /* Arrière-plan bleu au survol */
    color: #ffff; /* Texte en blanc au survol */
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>890</y>
     <width>145</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /*padding: 10px 20px;*/
    padding: 5px;
    font-size: 16px;
    text-align: center;
    text-decoration: none;
    border: 2px solid #792382;
    color: #792382;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: #43aff7;
}


</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseH">
   <property name="geometry">
    <rect>
     <x>1860</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-image: url(&quot;images/x.svg&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #800080; /* Couleur du texte violette */
    font-size: 20px; /* Taille de la police */
    font-weight: bold; /* Police en gras */
    background-color: #ffffff; /* Couleur de fond */
    border: 2px solid #800080; /* Bordure violette */
    border-radius: 5px; /* Coins arrondis */
    padding: 10px; /* Marge intérieure */
}

QLabel:hover {
    background-color: #add8e6; /* Couleur de fond au survol (bleu clair) */
    border-color: #4B0082; /* Couleur de la bordure au survol */
}</string>
   </property>
   <property name="text">
    <string>Ajouter un Oeuvre</string>
   </property>
  </widget>
  <zorder>CloseH</zorder>
  <zorder>textEdit</zorder>
  <zorder>Menu</zorder>
  <zorder>statOFF</zorder>
  <zorder>widget</zorder>
  <zorder>label</zorder>
  <zorder>qrcode</zorder>
  <zorder>logout</zorder>
  <zorder>frame</zorder>
  <zorder>formLayoutWidget_2</zorder>
  <zorder>historique</zorder>
  <zorder>label_3</zorder>
  <zorder>label_20</zorder>
  <zorder>label_22</zorder>
  <zorder>label_21</zorder>
  <zorder>downloadPDF</zorder>
  <zorder>label_5</zorder>
  <zorder>label_23</zorder>
  <zorder>search</zorder>
  <zorder>tableW</zorder>
  <zorder>triCombo</zorder>
  <zorder>label_2</zorder>
  <zorder>stat</zorder>
  <zorder>refresh</zorder>
  <zorder>label_24</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
